lib/fixtures/keycloak.go

```diff
@@ -120,6 +121,9 @@ func SetupKeycloak(ctx context.Context, kcConnectParams KeycloakConnectParams) e
 
 	opentdfClientID := "opentdf"
 	opentdfSdkClientID := "opentdf-sdk"
+	opentdfSdkCertExchangeClientID := "opentdf-sdk-cert-exchange"
```

Comment:
create separate client for cert exchange, i was having issues with opentdf-sdk since you cant do both token exchange and cert exchange